src/reference.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @copyright Copyright (c) 2023 Julien Veyssier <julien-nc@posteo.net>
+ *
+ * @author Julien Veyssier <julien-nc@posteo.net>
+ *
+ * @license AGPL-3.0-or-later
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { registerWidget } from '@nextcloud/vue-richtext'
+import Vue from 'vue'
+import VTooltip from '@nextcloud/vue/dist/Directives/Tooltip.js'
+import PageReferenceWidget from './views/PageReferenceWidget.vue'
+
+Vue.directive('Tooltip', VTooltip)
```

Comment:
No need for custom tooltips anymore ;) there was the general decision to move to browser native ones for accessibility

Reply:
:+1: VTooltip is not even used in the widget...